mixer/adapter/cloudwatch/operatorconfig/cloudwatch.yaml

```diff
@@ -2,31 +2,29 @@
 apiVersion: "config.istio.io/v1alpha2"
 kind: metric
 metadata:
-  name: requestcount
-  namespace: istio-system
+ name: requestcount
+ namespace: istio-system
 spec:
-  value: "1"
-  dimensions:
-    source_cluster: source.labels["cluster"] | "unknown"
-    source_service: source.labels["service"] | "unknown"
-    source_version: source.labels["version"] | "unknown"
-    destination_cluster: destination.labels["cluster"] | "unknown"
-    destination_service: destination.labels["service"] | "unknown"
-    destination_version: destination.labels["version"] | "unknown"
-    response_code: response.code | 200
-  monitored_resource_type: '"UNSPECIFIED"'
+ value: "1"
+ dimensions:
+   source_service: source.service | "unknown"
+   source_version: source.labels["version"] | "unknown"
+   destination_service: destination.service | "unknown"
```

Comment:
change it to destination.service.host while here, or I can do it later...

Reply:
If I end up pushing more commits, I'll fix. Good catch.

Reply:
done